spec/beaker/host/cisco_spec.rb

```diff
@@ -14,10 +14,10 @@ module Cisco
 
     describe '#prepend_commands' do
 
-      context 'for cisco-5' do
+      context 'for cisco-nexus' do
 
         before :each do
-          @platform = 'cisco-5-x86'
+          @platform = 'cisco-nexus-x86'
```

Comment:
@glennmatthews Won't the prepend commands be the same for nexus models/versions though? If this changes to include version names then you'll need separate context for n9k = 7.0(3)I2(1), 7.0.(3)I2(2), n7k = 7.3(0)D1, n5/6k = 6.0..., etc, which then needs to be updated with every release?